README.md

```diff
@@ -99,131 +99,109 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+ - Fixed the path where DSC module files were installed for the user-scope installer. (This was a hotfix for 0.82)
 
-### Peek
+### Mouse Without Borders
 
- - Prevent activating Peek when the user is renaming a file. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Added support to preview special folders like Recycle Bin and My PC instead of throwing an error.
- - Fixed a crash caused by double releasing a COM object from the module interface.
+ - Disabled non supported options in the old Mouse Without Borders UI. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added new GPO policies to block the use of some features. Thanks [@htcfreek](https://github.com/htcfreek)!
 
-### Power Rename
+### Peek
 
- - Improved apostrophe character handling for the Capitalize and Titlecase renaming flags. Thanks [@anthonymonforte](https://github.com/anthonymonforte)!
- - Added a feature to allow using sequences of random characters or UUIDs when renaming files. Thanks [@jhirvioja](https://github.com/jhirvioja)!
+ - Allow copying from the right-click menu in Dev files and Markdown previews.
 
 ### PowerToys Run
 
- - Improved the plugin descriptions for consistency in the UI. Thanks [@HydroH](https://github.com/HydroH)!
- - Fixed UI scaling for different dpi scenarios.
- - Fixed crash on a racing condition when updating UWP icon paths in the Program plugin. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed PowerToys Run hanging when trying to close an unresponsive window in the WindowWalker plugin. Thanks [@GhostVaibhav](https://github.com/GhostVaibhav)!
- - Fixed the example in the UnitConverter description to reduce confusion with the inches abbreviation (now uses "to" instead of "in"). Thanks [@acekirkpatrick](https://github.com/acekirkpatrick)!
- - Brought the acrylic background back and applied a proper fix to the titlebar accent showing through transparency.
- - Fixed an issue causing the transparency from the UI disappearing after some time.
+ - Fixed a crash on Windows 11 build 22000. (This was a hotfix for 0.82)
+ - Blocked a transparency fix code from running on Windows 10, since it was causing graphical glitches. (This was a hotfix for 0.82)
+ - Accept speed abbreviations like kilometers per hour (kmph) in the Unit Converter plugin. Thanks [@GhostVaibhav](https://github.com/GhostVaibhav)!
+ - Added settings to configure behavior of the "First week of year" and "First day of week" calculations in the DateTime plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Fixed wrong initial position of the PowerToys Run when switching between monitors with different dpi values.
+ - Started allowing interchangeable use of / and \ in the registry plugin paths.
+ - Added support to automatic sign-in after rebooting with the System plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added suggested use example results to the Value Generator plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Comment:
Not my work. Only did the review.

Reply:
My bad... Some copy paste and then forgot to change the user. Pushed a correction. Thanks for the review!